src/utils/astropy_util.py

```diff
@@ -0,0 +1,29 @@
+from astropy.time import Time
+from astropy.coordinates import get_sun, get_moon, CartesianRepresentation
+from astropy.constants import G, M_sun, M_earth
+from typing import Tuple, Optional
+from utils.constants import BodyEnum
+
+
+# Heavily reference the get_ephemeris function here: https://github.com/Cislunar-Explorers/FlightSoftware/blob/master/OpticalNavigation/core/observe_functions.py
+def get_ephemeris(observeStart: float, body: BodyEnum) -> Tuple[float, float, float]:
+    # Astropy needs unix timestamp in seconds!!!
+    current_time = observeStart
+    observeStart = observeStart - 11.716
```

Comment:
Why do we subtract 11.716 seconds here?

Reply:
Good point, I saw this from the OpNav code but didn't get rid of it. Will delete.